plugins/test/noop/main.go

```diff
@@ -123,7 +155,36 @@ func cmdDel(args *skel.CmdArgs) error {
 	return debugBehavior(args, "DEL")
 }
 
+func saveStdin() ([]byte, error) {
+	// Read original stdin
+	stdinData, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
+		return nil, err
+	}
+
+	// Make a new pipe for stdin, and write original stdin data to it
+	r, w, err := os.Pipe()
+	if err != nil {
+		return nil, err
+	}
+	if _, err := w.Write(stdinData); err != nil {
+		return nil, err
+	}
+	if err := w.Close(); err != nil {
+		return nil, err
+	}
+
+	os.Stdin = r
+	return stdinData, nil
+}
```

Comment:
@rosenhouse I think the issue here is that we need an *os.File to assign back to stdin, which bytes.NewReader() doesn't give us...